### PR TITLE
Add instructions for running the UI tests.

### PR DESCRIPTION
**Pull request by dgnball**
Changed **request_data.py** and updated readme instructions to make running the UI tests easier for new developers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,4 +91,47 @@ https://github.com/uktrade?q=spire
...
+* lite-api (port=**8100**)
+* casework (port=**8200**)
+* exporter (port=**8300**)
+
+You will also need to seed the lite-api database with the TEST_SSO_EMAIL (as well as running **seedall** 
+command - see https://github.com/uktrade/lite-api). If the email was `fake@fake.com` the 
+command would be (run against the lite-api virtual environment):
+
+```bash
+EXPORTER_USERS='[{"email"=>"fake@fake.com"}]' ./manage.py seedexporterusers
```
Comment (saruniitr) [inline README.md:130]: Not sure this is really required as we create test exporter user during setup

File: README.md [hunk 2]
```diff
@@ -91,4 +91,47 @@ https://github.com/uktrade?q=spire
...
+```
+
+Finally, to run a UI test on the command line (the exporter **login_test**):
+
+```bash
+PIPENV_DOTENV_LOCATION=exporter.env ENVIRONMENT=local pipenv run pytest -m "login_test" ui_tests/exporter
```
Comment (saruniitr) [inline README.md:136]: For this to work the scenario need to be tagged with `@login_test` otherwise no test will be selected.

**Review by saruniitr — APPROVED** [review]
few minor comments otherwise looks good to me.

File: README.md (hunk 1 above)
Comment (dgnball) [reply README.md]: Changed to internal users.

File: README.md (hunk 2 above)
Comment (dgnball) [reply README.md:137]: Mentioned the tag

**Pull request merged**